config/webpack.static.config.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+module.exports = {
+  output: {
+    filename: 'static/index.js',
+    library: 'paragon',
+    libraryTarget: 'umd',
+  },
+  plugins: [
+    new ExtractTextPlugin('static/paragon.min.css'),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.scss|\.css$/,
+        use: ExtractTextPlugin.extract({
+          use: [
+            {
+              loader: 'css-loader',
+              options: {
+                modules: true,
+                localIdentName: 'paragon__[local]',
+                sourceMap: true,
```

Comment:
Shouldn't there also be [`minimize: true`](https://github.com/webpack-contrib/css-loader#minimize) here so the CSS is actually minimized?

Reply:
Good catch!